spec/game_spec.rb

```diff
@@ -10,34 +10,52 @@ def initialize
   def output(message)
     @messages << message
   end
+
+  def gets_input
+    1
+  end
 end
 
 class TestDisplay
-  def present
-    'board'
+  def present(board_values)
+    board_values
+  end
+end
+
+class TestPrompt
+  def choose_a_space
+    'choose a space'
+  end
+end
+
+class TestBoardManager
+  attr_accessor :board_values
+
+  def initialize
+    @board_values = 'empty board'
+  end
+
+  def mark_space(_token, _space)
+    @board_values = 'board with one X mark'
   end
 end
 
-describe 'Tic Tac Toe Game' do
+describe 'Game' do
   context '#run' do
     it 'plays the game' do
       console = TestConsole.new
       welcome_message = 'Hello player!'
       display = TestDisplay.new
+      prompt = TestPrompt.new
+      board_manager = TestBoardManager.new
 
-      game = Game.new(console, welcome_message, display)
+      game = Game.new(console, welcome_message, display, prompt, board_manager)
 
       game.run
 
-      expected_messages = [welcome_message, 'board']
-
-      expect(game.console.messages).to eq(expected_messages)
-
-      # How to build test double to make assertions about how the game class will
-      # interact with testconsole object? Assert on that instead.
-      # Log string pattern in TDD by Example (could be array).
+      expected_output = [welcome_message, 'empty board', 'choose a space', 'board with one X mark']
 
-      # Build and test real console that you're going to inject in main (bin/run)
+      expect(game.console.messages).to eq(expected_output)
```

Comment:
Nice use of the "log string" pattern!

I would suggest that you avoid accessing the console through the game though. Do you want `.console` to be part of the public interface of the `Game`? An alternative would something like 

```suggestion
      expect(console.messages).to eq(expected_output)
```